wordpress-mobile/WordPress-Android
# Stats traffic remove today card

Fixes #20241 

-----

## To Test:

Test 1:
- Enable `stats_traffic_tab` feature flag (Me -> Debug settings - Remote Features)
- **Restart** the app
- Go to Stats
- `Verify` only Traffic tab, and Insights tabs are shown 
- `Switch` to Insights tab if not on it already
- `Tap` on Insights settings (⚙️)
- `Verify` there's no **Today Stats** option under General
- `Verify` that Today card is not shown

Test 2:
- Disable `stats_traffic_tab` feature flag
- **Restart** the app
- Go to Stats
- `Switch` to Insights tab if not on it already
- `Tap` on Insights settings (⚙️)
- `Verify` there's **Today Stats** option under General
- `Enable` **Today Stats** by tapping on it, and press Save
- Return to Insights
- `Verify` that Today card is shown


-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A 

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    -  Checked other places like My Site, Traffic, and Widget  where Today card is shown 

3. What automated tests I added (or what prevented me from doing so)

    - Existing unit tests

-----

## PR Submission Checklist:
